crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -0,0 +1,57 @@
+extern crate console_error_panic_hook;
+use std::rc::Rc;
+
+use bitwarden::{Client, ClientSettings};
+use log::{set_max_level, Level};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub enum LogLevel {
+    Trace,
+    Debug,
+    Info,
+    Warn,
+    Error,
+}
+
+fn convert_level(level: LogLevel) -> Level {
+    match level {
+        LogLevel::Trace => Level::Trace,
+        LogLevel::Debug => Level::Debug,
+        LogLevel::Info => Level::Info,
+        LogLevel::Warn => Level::Warn,
+        LogLevel::Error => Level::Error,
+    }
+}
+
+// Rc<...> is to avoid needing to take ownership of the Client during our async run_command
```

Comment:
question/issue?: We don't have a `run_command` anymore?

Reply:
Nop, but copy pasta until we figure out if we need RC here or not :shrug: